Description: Mark the pc file as data-only.
Author: Olivier Tilloy <olivier.tilloy@canonical.com>
Forwarded: https://gitlab.freedesktop.org/xdg/shared-mime-info/-/merge_requests/92

--- a/data/meson.build
+++ b/data/meson.build
@@ -38,5 +38,6 @@ pkgconfig.generate(
     name: 'shared-mime-info',
     description: 'Freedesktop common MIME database',
     version: meson.project_version(),
+    dataonly: true,
     install_dir: get_option('datadir') / 'pkgconfig'
 )
--- a/meson.build
+++ b/meson.build
@@ -2,7 +2,7 @@
 project('shared-mime-info',
     'c',
     version: '2.0',
-    meson_version: '>=0.49.0'
+    meson_version: '>=0.54.0'
 )
 
 config = configuration_data()
--- a/shared-mime-info.pc.in
+++ /dev/null
@@ -1,8 +0,0 @@
-prefix=@prefix@
-
-Name: shared-mime-info
-Description: Freedesktop common MIME database
-Version: @VERSION@
-Requires:
-Libs:
-Cflags:
